### PR TITLE
New style

### PR DESCRIPTION
**Pull request by MaxDrozdov**

### CODE REVIEW

File: js/app.service.date.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
+(function(){
+
+    angular.module('base.app').service('DateService', function(){
+
+        this.createDate = function(){
+            var stringDate = '';
+            var d = new Date();
+
+            switch (d.getDay()){
+                case 0: {
```
Comment (NE05KY) [inline js/app.service.date.js]: It's overengineering, use simple array instead of switch.

File: js/app.service.date.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                    break;
+                }
+                case 6: {
+                    stringDate += 'Saturday,';
+                    break;
+                }
+            }
+
+            switch (d.getMonth()){
+                case 0: {
```
Comment (NE05KY) [inline js/app.service.date.js]: same

File: src/css/style.news.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.newsBlock {
+    font-family: "Helvetica Neue";
```
Comment (NE05KY) [inline src/css/style.news.css]: Do you inject this font into your project?

File: js/app.service.date.js (hunk 1 above)
Comment (MaxDrozdov) [inline js/app.service.date.js]: Done.

File: js/app.service.date.js (hunk 2 above)
Comment (MaxDrozdov) [inline js/app.service.date.js]: Done.

File: src/css/style.news.css (hunk 3 above)
Comment (MaxDrozdov) [inline src/css/style.news.css]: Done.

**Pull request merged**